build/FrameworkPackage/MakeFrameworkPackage.ps1

```diff
@@ -259,6 +263,11 @@ Write-Host $makepriNew
 cmd /c $makepriNew
 if ($LastExitCode -ne 0) { Exit 1 }
 
+$makepriNew = "`"" + (Join-Path $WindowsSdkBinDir "makepri.exe") + "`" new /pr $fullOutputPath /cf $priConfigPath /of $priCBSOutputPath /in Microsoft.UI.Xaml.CBS /o"
```

Comment:
You can run this script locally, you should do that instead of iterating slowly on the pipelines.

Reply:
I have problem to compile the project locally. I always hit error and I didn't resolve it yet.

c1xx : fatal error C1076: compiler limit: internal heap limit reached [D:\repo\microsoft-ui-xaml\dev\dll\Microsoft.UI.X
aml.vcxproj]
c1xx : error C3859: Failed to create virtual memory for PCH [D:\repo\microsoft-ui-xaml\dev\dll\Microsoft.UI.Xaml.vcxpro
j]